examples/01-basic/11-removing-block-option/README.md

```diff
@@ -0,0 +1,7 @@
+# Removing Default Blocks Options
```

Comment:
Let's name it `removing-default-blocks` and title "Removing default (built-in) blocks".

Also, the paragraph below has some typos + maybe you can clarify which blocks you're removing